client/service/src/builder.rs

```diff
@@ -205,12 +205,13 @@ pub type TLightClientWithBackend<TBl, TRtApi, TExecDisp, TBackend> = Client<
 	TRtApi,
 >;
 
-/// Construct and hold different layers of Keystore wrappers
-pub struct KeystoreContainer {
-	keystore: Arc<dyn CryptoStore>,
-	sync_keystore: SyncCryptoStorePtr,
+enum KeystoreContainerInner {
```

Comment:
Why do we need to make an extra inner?

Reply:
Otherwise the enum data would be directly exposed. I think that's not something we want.